### PR TITLE
hl/readme update

### PR DESCRIPTION
**Pull request by sachincool**
- **chore(aws): remove unused tfy.tfvars file to clean up configuration files**
- **docs(README.md): update AWS setup guide and add Azure and GCP setup guides**

### CODE REVIEW

File: aws/README.md [hunk 1]
```diff
@@ -18,45 +18,60 @@ This repository contains Terraform configurations to set up the Truefoundry infr
...
 
    ```bash
-   aws s3api create-bucket --bucket your-terraform-state-bucket-name --region your-aws-region
-   aws s3api put-bucket-versioning --bucket your-terraform-state-bucket-name --versioning-configuration Status=Enabled
+   aws s3api create-bucket --bucket <bucket-name> --region <region>
+   aws s3api put-bucket-versioning --bucket <bucket-name> --versioning-configuration Status=Enabled
    ```
 
-2. **Update `backend.tf`**
+3. **Update `backend.tf`**
```
Comment (shubhamrai1993) [inline aws/README.md:51]: Clone repository should happen before this

File: aws/README.md [hunk 2]
```diff
@@ -74,88 +89,119 @@ Additionally, you need:
    terraform init
    ```
 
-3. **Apply Network Module**
+3. **Plan and Apply Network Module**
```
Comment (shubhamrai1993) [inline aws/README.md:92]:
```suggestion
3. **Run terraform plan for the complete module**
```

File: aws/README.md [hunk 3]
```diff
@@ -74,88 +89,119 @@ Additionally, you need:
...
-2. **Security Groups**: Check for lingering deletions.
-3. **Persistent Volumes**: Not automatically removed, manual deletion required for:
-   - ArgoCD resources (Grafana, Kubecost, Truefoundry, Prometheus, Loki)
-4. **EBS Volumes**: Check and delete manually.
-5. **Endpoints**: Manually delete EFS and GuardDuty endpoints.
+1. **Karpenter Nodes**
+   - Manual deletion may be required
+   - Check EC2 instances and ASGs
+
+2. **Security Groups**
```
[the remark this answers is not in the document]
Comment (sachincool) [reply aws/README.md:165]: security groups which are attached to nodes would not removed till we destroy all instances

[the remark this answers is not in the document]
Comment (sachincool) [reply aws/README.md:165]: if we're deleting karpenter nodes and pvcs , It should clean out everything. terraform destroy should succeed

File: aws/README.md [hunk 4]
```diff
@@ -74,88 +89,119 @@ Additionally, you need:
...
+
+5. **VPC Endpoints**
+   Manual cleanup needed for:
+   - EFS endpoints
+   - GuardDuty endpoints
+   - S3 endpoints
 
 ## Next Steps
 
 After successful installation:
```
[the remark this answers is not in the document]
Comment (sachincool) [reply aws/README.md:186]: links for Grafana/ prometheus setup? 
no,  that reminds me  Links for DNS and loadbalancer is missing, will add that too

**Review by shubhamrai1993 — APPROVED** [review]

**Review by shubhamrai1993 — APPROVED** [review]

**Pull request merged**